src/assets/_project/_blocks/components/accordion/_qg-accordion.scss

```diff
@@ -170,7 +170,7 @@
     }
   }
   .collapsing-section {
-    background: #fff;
+    background: white;
```

Comment:
so why convert from hex to word? #fff to white. why not a variable then?

Reply:
It's just less of a magic string.